doc/en/customize.rst

```diff
@@ -99,73 +185,36 @@ Example:
     pytest path/to/testdir path/other/
 
 will determine the common ancestor as ``path`` and then
-check for ini-files as follows:
+check for configuration files as follows:
 
 .. code-block:: text
 
     # first look for pytest.ini files
     path/pytest.ini
-    path/tox.ini    # must also contain [pytest] section to match
-    path/setup.cfg  # must also contain [tool:pytest] section to match
+    path/pyproject.toml  # must contain a [tool.pytest.ini_options] table to match
+    path/tox.ini         # must contain [pytest] section to match
+    path/setup.cfg       # must contain [tool:pytest] section to match
     pytest.ini
-    ... # all the way down to the root
+    ... # all the way up to the root
 
     # now look for setup.py
     path/setup.py
     setup.py
-    ... # all the way down to the root
-
-
-.. _`how to change command line options defaults`:
-.. _`adding default options`:
-
-
-
-How to change command line options defaults
```

Comment:
Moved to `example/simple.rst`, seems to fit there better; here was out of place.